src/components/RichTextEditor/Plugins/ButtonPlugin/components/FloatingButton/InsertButtonModal/InsertButtonModal.tsx

```diff
@@ -4,18 +4,12 @@ import React, { CSSProperties, ReactElement, ReactNode, useState } from 'react';
 
 import { FormControl } from '@components/FormControl';
 import { useInsertModal } from './useInsertModal';
-import { useRichTextEditorContext } from '@components/RichTextEditor/context/RichTextEditorContext';
 import { InsertModal } from '@components/RichTextEditor/Plugins/LinkPlugin/FloatingLink/InsertLinkModal';
-import { ButtonStyles } from '@components/RichTextEditor/Plugins/TextStylePlugin/TextStyles';
+import { getButtonStyleCssProperties } from '@components/RichTextEditor/Plugins/helper';
```

Comment:
Sometimes the import for `getTextStyleCssProperties` is like this, sometimes it's `from '../../'` and sometimes it's `from '../../helper'`. Maybe keep it consistent. :)

Reply:
Adjusted it so its always via the `helper`